README.md

```diff
@@ -1 +1,60 @@
-#spring-boot-remote-git-autoconfigure
+spring-boot-remote-git-autoconfigure
+=======================================
+It can download the Spring Boot configurations from remote git repo and load the configuration to Spring boot APP
+
+## Prerequisites
+You should install the git client successfully and the configure the ssh auto verify with remote git repo successfully. It mean that you can use the OS git command pull the remote configuration repo item successfully.
+
+## Add the dependency jar 
+  maven:
+```
+<dependency>
+    <groupId>com.vmware.singleton</groupId>
+    <artifactId>spring-boot-remote-git-autoconfigure</artifactId>
+    <version>{version}</version>
+</dependency>
+
+```
+  gradle:
+```
+implementation 'com.vmware.singleton:spring-boot-remote-git-autoconfigure:{version}'
+
+```
+
+## create the remote git config repository for Spring Boot apps 
```

Comment:
Is it better move it to 'Prerequisites' part?

Reply:
It is part of how to use it.